_ingest-pipelines/processors/split.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: default
+title: Split
+parent: Ingest processors
+nav_order: 270
+---
+
+# Split processor
+
+The `split` processor is used to split a string field into an array of substrings based on a specified delimiter.
+
+The following is the syntax for the `split` processor:
+
+```json
+{
+  "split": {
+    "field": "field_to_split",
+    "separator": "delimiter",
+    "target_field": "split_field"
+  }
+}
+```
+{% include copy-curl.html %}
+
+## Configuration parameters
+
+The following table lists the required and optional parameters for the `split` processor.
+
+Parameter | Required/Optional | Description |
+|-----------|-----------|-----------|
+`field` | Required | The field containing the string to be split. Supports [template snippets]({{site.url}}{{site.baseurl}}/ingest-pipelines/create-ingest/#template-snippets).
+`separator` | Required | The delimiter used to split the string. This can be a regular expression pattern.
+`target_field` | Required | The field where the array of substrings is stored. Supports [template snippets]({{site.url}}{{site.baseurl}}/ingest-pipelines/create-ingest/#template-snippets).
+`ignore_missing` | Optional	| Specifies whether the processor should ignore documents that do not contain the specified 
+field. If set to `true`, the processor ignores missing values in the field and leaves the `target_field` unchanged. Default is `false`. 
+`override_target` | Optional | Determines what happens when `target_field` exists in the document. If set to `true`, the processor overwrites the existing `target_field` value with the new value. If set to `false`, the existing value remains and the processor does not overwrite it. Default is `false`.
+`description` | Optional | A brief description of the processor.
+`if` | Optional | A condition for running the processor.
+`ignore_failure` | Optional | Specifies whether the processor continues execution even if it encounters an error. If set to 
+`true`, failures are ignored. Default is `false`.
+`on_failure` | Optional | A list of processors to run if the processor fails.
+`tag` | Optional | An identifier tag for the processor. Useful for debugging in order to distinguish between processors of the same type.
+
```

Comment:
Another optional parameter `preserve_trailing` is missing, the parameter defaults to `false`, when set to true, the processor will remove the empty trailing field like '' from the resulting array.